content/22.vaccines.md

```diff
@@ -244,6 +244,14 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Inactivated Whole Virus Vaccines
 
 <!--Rephrase to match figure? Or rephrase figure?-->
+This type of vaccine uses a "killed" full viral particle to prime the immune system.
+The virus is killed using formaldehyde or &beta;-propiolactone [@doi:10.1080/14760584.2017.1357471].
+The size of the virus particle makes it ideal for uptake by antigen presenting cells (APC), which leads to stimulation of helper T-cells [@doi:10.1038/nri2868].
+Additionally, the array of epitopes on the surface of the virus increases antibody binding efficiency [@doi:10.1038/nri2868].
+The native conformation of the surface proteins, which is also important for eliciting an immune response, is preserved using this technique.
+Membrane proteins, which support B-cell response to surface proteins, are also included using this method [@doi:10.1084/jem.164.4.1114]. 
+Overall, the vaccine is able to mimic the key properties of the virus that stimulate a robust immune response, but the risk of adverse reactions is reduced because the virus is dead.
```

Comment:
```suggestion
Overall, these vaccines are able to mimic the key properties of the virus that stimulate a robust immune response, but the risk of adverse reactions is reduced because the virus is inactivated, thus unable to replicate.
```